src/parser.cpp

```diff
@@ -1150,6 +1159,19 @@ void Parser::ParserImpl::loadImport(const ImportSourcePtr &importSource, const M
         attribute = attribute->next();
     }
     XmlNodePtr childNode = node->firstChild();
+
+    if (!childNode) {
+        auto issue = Issue::create();
+        if (id.empty()) {
+            issue->setDescription("Import from '" + node->attribute("href") + "' is empty. It will not be loaded into the model.");
+        } else {
+            issue->setDescription("Import from '" + node->attribute("href") + "' has an id of '" + id + "' but is empty. It will not be loaded into the model, and the associated id bookmark will be lost.");
```

Comment:
as above, wouldn't use the word bookmark.

Reply:
Removed in cc44b080